app/api/v2/models/oauth.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
```

Comment:
This import is breaking FastApi's `depends`. Related issue: https://github.com/tiangolo/fastapi/issues/1654